### PR TITLE
LWG Poll 19: P0586R2 Safe integral comparisons

### LINKED ISSUE #3721: [2020-02 LWG Poll 19] P0586R2 Safe integral comparisons
with the change of the name of the feature test macro from __cpp_lib_cmp_equal to __cpp_lib_integer_comparison_functions. 

resolves DE208

### PR DESCRIPTION
**Pull request by burblebee**
Also fixes NB DE 208 (C++20 CD).

Fixes #3721. 
Fixes cplusplus/papers#259.
Fixes cplusplus/nbballot#206.

### CODE REVIEW

File: source/utilities.tex [hunk 1]
```diff
@@ -75,6 +75,25 @@
   // \ref{declval}, declval
   template<class T>
     add_rvalue_reference_t<T> declval() noexcept;   // as unevaluated operand
+
+  // \ref{utility.safeintcmp}, Safe integral comparisons
```
Comment (johelegp) [inline source/utilities.tex]:
```suggestion
  // \ref{utility.safeintcmp}, safe integral comparisons
```

File: source/support.tex [hunk 2]
```diff
@@ -583,6 +583,7 @@
 #define @\defnlibxname{cpp_lib_chrono}@                            201907L // also in \libheader{chrono}
 #define @\defnlibxname{cpp_lib_chrono_udls}@                       201304L // also in \libheader{chrono}
 #define @\defnlibxname{cpp_lib_clamp}@                             201603L // also in \libheader{algorithm}
+#define @\defnlibxname{cpp_lib_cmp_equal}@                         202002L // also in \libheader{utility}
```
Comment (timsong-cpp) [inline source/support.tex]: Per the motion, the feature-test macro should be `__cpp_lib_integer_comparison_functions`.

File: source/utilities.tex (hunk 1 above)
Comment (timsong-cpp) [reply source/utilities.tex]: We should not use "safe".

Comment (jwakely) [reply source/utilities.tex]: And we should not use "integral" either.

I suggest "integer comparison functions" instead.

Comment (burblebee) [reply source/utilities.tex]: changed to "integer comparison functions"

File: source/support.tex (hunk 2 above)
Comment (burblebee) [reply source/support.tex]: done

File: source/utilities.tex [hunk 3]
```diff
@@ -75,6 +75,25 @@
   // \ref{declval}, declval
   template<class T>
     add_rvalue_reference_t<T> declval() noexcept;   // as unevaluated operand
+
+  // \ref{utility.safeintcmp}, integer comparison functions
```
Comment (timsong-cpp) [inline source/utilities.tex]: Change the stable name too?

Comment (burblebee) [reply source/utilities.tex]: renamed to utility.intcmp

**Pull request merged**